modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/Link.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayIcon from '@clayui/icon';
+import ClayLink from '@clayui/link';
+import classNames from 'classnames';
+import React from 'react';
+
+export default function Link({
+	additionalProps: _additionalProps,
+	componentId: _componentId,
+	cssClass,
+	icon,
+	label,
+	locale: _locale,
+	portletId: _portletId,
+	portletNamespace: _portletNamespace,
+	...otherProps
+}) {
+	return (
+		<ClayLink className={cssClass} {...otherProps}>
+			{otherProps.title && (
```

Comment:
This doesn't match with the markup generated by the tag (I don't see any `<span sronly>` in the tag). Anyway, if we're adding this because accessibility, this should be implemented in Clay, not here, IMHO (and after adding it in clay update the tag SSR markup).

Reply:
Ok I will remove it, I just added there for test.